issue/67-Docu. Special-Workflow for Containers on Remote Servers (#108)
nautobot/nautobot: nautobot/docs/development/getting-started.md

To start Nautobot, select "Run Without Debugging" or "Start Debugging" from the Run menu. Once Nautobot has started, you will be prompted to open a browser to connect to Nautobot.

#### Special Workflow for Containers on Remote Servers

A slightly different workflow is needed when the container is running on a ssh connected server. VScode will not offer the "Reopen in Container" option on a remote connected server.

> After `invoke build` use docker-compose to start the containers. This prevents that the http service is starting inside the container.

Reviewer: ```suggestion
After `invoke build` use `docker-compose` as follows to start the containers. This prevents the HTTP service from automatically starting inside the container.
```
